### PR TITLE
feat(event): onOneServer support

### LINKED ISSUE #39: feature request: onOneServer
https://laravel.com/docs/5.7/scheduling#running-tasks-on-one-server

### PR DESCRIPTION
**Pull request by zacksleo**
fixed #39

### CODE REVIEW

File: src/Event.php [hunk 1]
```diff
@@ -518,6 +518,20 @@ public function withoutOverlapping()
...
     }
 
+    /**
+     * Allow the event to only run on one server for each cron expression.
+     *
+     * @return $this
+     */
+    public function onOneServer()
+    {
+        if ($this->_mutex instanceof \yii\mutex\FileMutex) {
```
Comment (gietos) [inline src/Event.php]: Please, import classes

File: src/Event.php [hunk 2]
```diff
@@ -518,6 +518,20 @@ public function withoutOverlapping()
...
+            throw new \yii\base\InvalidConfigException("You must config mutex in the application component, except the FileMutex.");
```
Comment (gietos) [inline src/Event.php]: Please use single quotes for strings

**Comment by gietos** [comment]
thanks, @zacksleo

**Comment by brandonkelly** [comment]
When is this going to make it into a release?

**Comment by gietos** [comment]
Went to 1.1.2

**Pull request merged**